code/ATMOSPHERICS/he_pipes.dm

```diff
@@ -9,8 +9,9 @@
 	minimum_temperature_difference = 20
 	thermal_conductivity = OPEN_HEAT_TRANSFER_COEFFICIENT
 
-	var/RADIATION_CAPACITY = 32000       // Radiation isn't particularly effective (TODO BALANCE)
+	var/RADIATION_CAPACITY = 30000       // Radiation isn't particularly effective (TODO BALANCE)
```

Comment:
is there a reason this doesn't use the RADIATION_CAPACITY define which is also 30000?

also having a local var in all caps is bad practice

Reply:
There shouldn't be such a define, it's being undefined at the end of thermal_plates.dm which is the only file using it. As for why it's a var, only N3X15 knows I guess. In general it's probably better for these things to be vars (so it's easier to make more effective subtypes etc.), but pipecode right now *mostly* uses defines so I guess I'll make this a define too.

I also cleaned up the defines in this file and thermal_plates.dm, adding some undefs for defines that weren't used anywhere else.

Reply:
Oh, nevermind, looks like that didn't compile. Probably shouldn't use defines for H/E pipes because the manifolds have a different radiation capacity.

I'll make it a var for thermal plates as well for consistency's sake. Pipecode has a ton of shit that should be refactored to be either defines or vars.